db/migrate/20151021105451_create_prisons.rb

```diff
@@ -1,6 +1,6 @@
 class CreatePrisons < ActiveRecord::Migration
   def change
-    create_table :prisons do |t|
+    create_table :prisons, id: :uuid do |t|
```

Comment:
Sing with me now...sche-ma-less...Forward comrades!  To the brave new world of the schemaless key value store with relations!